### PR TITLE
Graph.json clean up

### PR DESCRIPTION
**Pull request by QilongTang**
### Purpose

Following https://github.com/DynamoDS/Dynamo/pull/7962. Addressing the remaining incorrect params found in Graph.JSON.

This PR:
1. Use JsonProperty to replace `Nodes` with `NodeViews` block
2. Added nodeId to each NodeViews block
3. Removed `LastModified`, `LastModifiedBy`, `NearPlaneDistance` and `FarPlaneDistance` within Cameras since they are not used in Reach or AVP. `IsCustomNode`, `ElementResolver`, `Notes` are preserved for now
4. Confirmed `FunctionSignature` property did get serialized for just ZeroTouchNode, nodeType is `FunctionNode` in this case

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@mjkkirschner @ramramps @ColinDayOrg 

### FYIs

@gregmarr @ikeough

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Core/SerializationExtensions.cs [hunk 1]
```diff
@@ -25,12 +28,54 @@ internal static string ToJson(this WorkspaceViewModel viewModel)
...
+                }
             };
 
             var json = JsonConvert.SerializeObject(viewModel, settings);
 
             return json;
         }
     }
+
+    public class NodeViewModelConverter : JsonConverter
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/Core/SerializationExtensions.cs]: This converter will be moved to a new file

File: src/DynamoCoreWpf/ViewModels/Core/SerializationExtensions.cs [hunk 2]
```diff
@@ -25,12 +28,54 @@ internal static string ToJson(this WorkspaceViewModel viewModel)
...
+
+            // Start a new object for all the nodeViewModel properties
+            writer.WriteStartObject();
+            writer.WritePropertyName("Name");
+            writer.WriteValue(nodeViewModel.Name);
+            writer.WritePropertyName("X");
+            writer.WriteValue(nodeViewModel.X);
+            writer.WritePropertyName("Y");
+            writer.WriteValue(nodeViewModel.Y);
+            writer.WritePropertyName("isVisible");
```
Comment (gregmarr) [inline src/DynamoCoreWpf/ViewModels/Core/SerializationExtensions.cs]: All properties in the schema start with a capital letter.

**Comment by ramramps** [comment]
I am fine with this change  LGTM @QilongTang

**Comment by QilongTang** [comment]
Verified no regression introduced. If no objections, merging in a few

**Comment by mjkkirschner** [comment]
LGTM

**Pull request merged**